<a href="https://colab.research.google.com/github/samhita13/Python101/blob/main/assignment_4.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [28]:
print("Budget Analysis")
BudgetGoal = int(input("Enter the amount you have bugeted $ for a month."))
print("Enter Expenses by type 1.Clothing 2.Food 3.Entertainment 4.Rent 5.Transport ")
numberofbudgets=1
totalexpense=0
while(numberofbudgets<=5):
 EXPENSE =int(input("Budget number " + str(numberofbudgets) + "  "))
 totalexpense=totalexpense+EXPENSE
 numberofbudgets=numberofbudgets+1
if (totalexpense<BudgetGoal):
  underexpense=BudgetGoal-totalexpense
  print("you are under budget by " + str(underexpense))
elif (totalexpense>BudgetGoal):
  overexpense= totalexpense-BudgetGoal
  print("you are over budget by " + str(overexpense)+ " $")
else:
  print("You have reacheched your budgeted goal!")


Budget Analysis
Enter the amount you have bugeted $ for a month.1300
Enter Expenses by type 1.Clothing 2.Food 3.Entertainment 4.Rent 5.Transport 
Budget number 1  100
Budget number 2  400
Budget number 3  100
Budget number 4  500
Budget number 5  200
You have reacheched your budgeted goal!


Budget Analysis
Write a program that asks the user to enter the amount that he or she has budgeted for a month.
A loop should then prompt the user to enter each of his or her expenses for the month and keep
a running total. When the loop finishes, the program should display the amount that the user is
over or under budget

In [15]:
print("Distance Traveled")
hours = 1
speedmph=int(input("What was the speed of your vehicle in miles per hour?"))
totaltime=int(input("How many hours have you traveled?"))
##distance=hours*totaltime
while (hours<=totaltime):
 distance=speedmph*hours
 print(str(hours)+" hours is " + str(distance))
 hours=hours+1


Distance Traveled
What was the speed of your vehicle in miles per hour?40
How many hours have you traveled?3
1 hours is 40
2 hours is 80
3 hours is 120


4. Distance Traveled
The distance a vehicle travels can be calculated as follows:
distan⁡ce = speed × time
For example, if a train travels 40 miles per hour for three hours, the distance traveled is 120
miles. Write a program that asks the user for the speed of a vehicle (in miles per hour) and the
number of hours it has traveled. It should then use a loop to display the distance the vehicle has
traveled for each hour of that time period. Here is an example of the desired output:
What is the speed of the vehicle in mph? 40
How many hours has it traveled? 3
Hour Distance Traveled
1 40
2 80
3 120


In [8]:
print("Celsius to Farenheight Table")
celsius=0
print("Celsius  Farenheight")
while (celsius<=20):
  farenheight=(9/5*celsius)+32
  print(" "+str(celsius)+"        "+ str(farenheight))
  celsius = celsius+1

Celsius to Farenheight Table
Celsius  Farenheight
 0        32.0
 1        33.8
 2        35.6
 3        37.4
 4        39.2
 5        41.0
 6        42.8
 7        44.6
 8        46.4
 9        48.2
 10        50.0
 11        51.8
 12        53.6
 13        55.400000000000006
 14        57.2
 15        59.0
 16        60.8
 17        62.6
 18        64.4
 19        66.2
 20        68.0


6. Celsius to Fahrenheit Table
Write a program that displays a table of the Celsius temperatures 0 through 20 and their
Fahrenheit equivalents. The formula for converting a temperature from Celsius to Fahrenheit is
F=95C+32
where F is the Fahrenheit temperature, and C is the Celsius temperature. Your program must
use a loop to display the table.